theano/tensor/tests/test_extra_ops.py

```diff
@@ -37,6 +38,80 @@ def test_cpu_contiguous():
                                             [numpy.random.rand(5, 7, 2)])
 
 
+class TestSearchsortedOp(utt.InferShapeTester):
+
+    def setUp(self):
+        super(TestSearchsortedOp, self).setUp()
+        self.op_class = SearchsortedOp
+        self.op = SearchsortedOp()
+
+        self.x = T.vector('x')
+        self.v = T.tensor3('v')
+
+        self.a = 30 * np.random.random(50).astype(config.floatX)
+        self.b = 30 * np.random.random((8, 10, 5)).astype(config.floatX)
+        self.idx_sorted = np.argsort(self.a)
+
+    def tearDown(self):
```

Comment:
I don't understand why this is needed. We don't do it elsewhere. Is there a particular reason you do this? I think by default all member get removed and Python gc would collect them afterward. But if you know they are needed, please explain. I never check my hypothesis.